CHANGELOG.md

```diff
@@ -6,6 +6,14 @@
 
 `dfx start --pocketic` is now compatible with `--artificial-delay` and the `subnet_type`  configuration option, and enables `--enable-canister-http` by default.
 
+### feat: Support canister log allowed viewer list
+
+Added support for the canister log allowed viewer list, which configures which users are allowed to read a canister's logs, with no need to be set as controller of the canister.
```

Comment:
NIT: rephrasing to avoid duplicated which-which

`Added support for the canister log allowed viewer list, enabling specified users to access a canister's logs without needing to be set as the canister's controller.`

Reply:
Fixed at https://github.com/dfinity/sdk/pull/3921/commits/201e2b727e6f000da193c446a0d68d4914eb3b0c.